lib/CBonusTypeHandler.cpp

```diff
@@ -94,7 +94,9 @@ std::string MacroString::build(const GetValue & getValue) const
 CBonusType::CBonusType()
 {
 	hidden = true;
-	icon = nameTemplate = descriptionTemplate = "";
+	icon.clear();
+    nameTemplate.clear();
+    descriptionTemplate.clear();

```

Comment:
we use tabs for indentation
```suggestion
	nameTemplate.clear();
	descriptionTemplate.clear();
```

Reply:
@kambala-decapitator fix it https://github.com/vcmi/vcmi/pull/1111/commits/2c01e9680c3778d5f4a2732eb6f7d01c42fc2501